src/components/Search/Search.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import classes from './search.scss';
+
+export default class Seach extends React.Component {
+
+  searchByKey(event) {
+    console.log(event.target.value);
+  }
+
+  render() {
+    return (
+      <section className={classes.search}>
+        <input
+          type="serch"
+          className={classes.input}
+          onChange={::this.searchByKey}
```

Comment:
Do not bind in `render() { ... }`. Please use fat arrow functions instead of bind().
https://medium.com/@esamatti/react-js-pure-render-performance-anti-pattern-fb88c101332f